brightwind/load/station.py

```diff
@@ -814,9 +824,9 @@ def __meas_point_merge(sensor_cfgs, sensors=None, mount_arrgmts=None):
     def __get_properties(self):
         meas_props = []
         for meas_point in self._meas_data_model:
-            # col_names_raised = _raise_child(meas_point, child_to_raise='column_name')
-            # sen_cfgs = _raise_child(col_names_raised, child_to_raise='sensor_config')
-            sen_cfgs = _raise_child(meas_point, child_to_raise='sensor_config')
+            logger_meas_cfgs = _raise_child(meas_point, child_to_raise='sensor_config')
```

Comment:
How does 'sensor_config' work in this line? There shouldn't be any 'sensor_config'.

Reply:
removed